webpack.config.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
Let's get rid of these `use strict` clauses as they aren't needed in the current version of Node